ZWaveLib/Utility/Utility.cs

```diff
@@ -121,23 +113,6 @@ public static byte[] ExtractRoutingFromBitMask(byte[] receivedMessage)
             }
             return routingInfo.ToArray();
         }
-
-        public static void DebugLog(DebugMessageType dtype, string message)
```

Comment:
I think it's better to incorporate NLog logging into this method rather than delete it because it's easier to read colored logs in console.